<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Program Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>10</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="automaticallyResizeLabel">
      <property name="text">
       <string>Automatically Resize to Fit Grid</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="automaticallyResizeCheckBox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="overrideWindowStyleLabel">
      <property name="text">
       <string>Override Window Style (requires restart)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="overrideWindowStyleCheckBox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="minimiseToTrayLabel">
      <property name="text">
       <string>Minimise to System Tray</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="minimiseToTrayCheckBox">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="maxHostsLabel">
      <property name="text">
       <string>Maximum Hosts Remembered</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="maxHostsSpinBox">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Disable Sound</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="disableSound">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Show Connect on Startup</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="connectStartup">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Use webserver as backup connection</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="webBackup">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Webserver script:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLineEdit" name="serverScript">
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
